src/lib/impl/PublisherImpl.js

```diff
@@ -241,7 +241,7 @@ class PublisherImpl extends EventEmitter {
         if (this.getLatching()) {
           this._lastSentMsg = serializedMsg;
         }
-        msgCount++;
+        msgCount = msgCount > 253 ? 0 : msgCount++;
```

Comment:
why does this rollover at 253?

Reply:
Using this expression the maximum value that we can reach is 254. I'm updating the expression to reach 255.
Btw the expression should be ( post increment will leave `msgCount` to 0 )
```
msgCount = msgCount > 253 ? 0 : ++msgCount;
```

Reply:
> why does this rollover at 253?

the rollover at 253 (better, at 254, see next commits) is necessary because in the udp header we use, the msgId is a uint8: see src/utils/udpros_utils.js line 283